toolchain/language_server/context.h

```diff
@@ -47,6 +52,11 @@ class Context {
     std::unique_ptr<Lex::TokenizedBuffer> tokens_;
     std::unique_ptr<Parse::Tree> tree_;
     std::unique_ptr<Parse::TreeAndSubtrees> tree_and_subtrees_;
+
+    // converts (line, row) position in source code to index
```

Comment:
```suggestion
    // Converts (line, row) position in source code to index.
```

As noted above, we use sentence punctuation in comments.